packages/repository-json-schema/src/build-schema.ts

```diff
@@ -15,6 +15,7 @@ import {
 import * as debugFactory from 'debug';
 import {JSONSchema6 as JSONSchema} from 'json-schema';
 import {JSON_SCHEMA_KEY, MODEL_TYPE_KEYS} from './keys';
+const nativeUtils = require('util');
```

Comment:
Please use ES6 imports.

```suggestion
import {inspect} from 'util';
```

Or if you want to import everything:

```suggestion
import * as nativeUtils from 'util';
```

Reply:
Changes updated